OctoAwesome/OctoAwesome/CodeExtensions/StringExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace OctoAwesome.CodeExtensions
+{
+    /// <summary>
+    /// String Extensions
+    /// </summary>
+    public static class StringExtensions
+    {
+        /// <summary>
+        /// Split a string.
+        /// </summary>
+        /// <param name="input">String to split</param>
+        /// <param name="options">Options of the Split operation</param>
+        /// <param name="separator">Separators</param>
+        /// <returns></returns>
+        public static string[] Split(this string input, StringSplitOptions options, params char[] separator)
+        {
+            return input.Split(separator, options);
+        }
+        /// <summary>
+        /// Checks if the string is NUll or Empty
+        /// </summary>
+        /// <param name="input">String to check</param>
+        /// <returns></returns>
+        public static bool IsNullOrEmpty(this string input)
+        {
+            return string.IsNullOrEmpty(input);
```

Comment:
Das ist unnötig das kann .NET nämlich von Haus aus von daher sehe ich die CodeExtension nicht als sinnvoll an. Vorallem da Extensions der grauen sind was Performance angeht.

Reply:
Da muss man aber immer string.IsNullOrEmpty(string) schreiben.
Echt hätte erwartet das der Jitter das inline macht...

Reply:
Nope die mergen das recht umständlich während der Laufzeit. Da ist zwar ne änderung geplant in C# 8 was ein extension keyword einfügt aber leider noch keine perfomance verbesserung. Um das performanter zu machen bedarf die .NET runtime in dem punkt ein redesign.

Außerdem ist sehr häufig string.IsNullOrWhitespace besser.

Reply:
Ich würde es durchgehen lassen wenn die anderen ihr grünes licht dazu geben.